### ISSUE TITLE
Call to undefined function UniSharp\LaravelFilemanager\Traits\str_singular() on Laravel 6

### ISSUE THREAD

**Issue by kastony** [UniSharp/laravel-filemanager#834]
I got this under laravel 6, any suggestions ?

 * Operating system : Mac
 * Laravel version : 6.0.3
 * Package version : 1.9.2

**Comment by gillidandaweb** [comment]
You can pull them in with `composer require laravel/helpers`

**Comment by Dimimo** [comment]
> You can pull them in with `composer require laravel/helpers`
That's true (for now) but not recommended. It takes 5 lines of code to update the project to the Str class, away from the helpers. I for one, have updated my project according to the 5.8 recommendations.

**Comment by stefandb** [comment]
add "composer require laravel/helpers" to your composer.json is a temp solution. This package needs a PR to fix this for the feature.

**Comment by adeguntoro** [comment]
so it mean the only way to fix this problem is going back to Laravel 5.8.x ?

**Comment by rjvandoesburg** [comment]
@adeguntoro no you don't need to downgrade, just add the `laravel/helpers` package to your application with `composer require laravel/helpers`.
But as Stefan says, this should be fixed within the package.

**Comment by M-kamal-Emira** [comment]
> 
> 
> so it mean the only way to fix this problem is going back to Laravel 5.8.x ?

run
composer require laravel/helpers

**Comment by fomvasss** [comment]
Why vendor (UniSharp) does not accept pull requests?! This packages do not correct work in Laravel 6!
[https://i.imgur.com/CrFW9eW.png](https://i.imgur.com/CrFW9eW.png)

**Comment by janooosh** [comment]
Any news on that? Can i just add these "5 lines" according to @Dimimo ?

**Comment by denibro** [comment]
> 
> 
> You can pull them in with `composer require laravel/helpers`

thank you..

**Comment by streamtw** [comment]
This issue should be no more in recent versions.
